bot.py

```diff
@@ -1,205 +1,351 @@
+import discord.ext.commands
+import discord
 import logging
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger('bot')
 
-import discord
-import discord.ext.commands
 
-BOT_TOKEN = 
-COMMAND_CHANNEL_ID = 
-ORGANIZER_ROLE_ID = 
+BOT_TOKEN = ''
+COMMAND_CHANNEL_ID = ''
+JAMMER_ROLE_ID = ''
+ADMIN_ROLE_ID = 
 TEAM_NAME_PREFIX = 'Team '
 
-client = discord.ext.commands.Bot(command_prefix='!', intents=discord.Intents.all())
+client = discord.ext.commands.Bot(
+    command_prefix='!', intents=discord.Intents.all())
+
 
 def build_category_name(team_name):
-	return f"🧩 {team_name}"
+    return f"🧩 {team_name}"
```

Comment:
Such a change expanding all tabs into spaces, touching almost every line of code, should be a commit in its own right and not grouped together with other changes.